libs/core/src/components/sage-icon/sage-icon.docs.stories.mdx

```diff
@@ -68,10 +67,9 @@ will render the following output:
   <sage-icon name="upload" size="2rem"></sage-icon>
 </Canvas>
 
-
 export const IconItems = (args) => (
-  Object.values(sageIconsJson["icons"]).map((icon) => (
-    <IconItem name={icon.name}>
+  Object.values(sageIconsJson["icons"]).map((icon, index) => (
+    <IconItem key={index} name={icon.name}>
```

Comment:
I understand why you did this, but to further suppress the warning you should do something below. Since react suggests the need for a key and not using an index as a key.
 
```
key=`item-${index}`
```